moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -73,24 +73,68 @@ QString subframe_poses_to_qstring(const moveit::core::FixedTransformsMap& subfra
 
 namespace moveit_rviz_plugin
 {
-void MotionPlanningFrame::importObjectFromFileButtonClicked()
+void MotionPlanningFrame::shapesComboBoxChanged(const QString& text)
 {
-  QString path = QFileDialog::getOpenFileName(this, tr("Import Object Mesh"), QString(),
-                                              "CAD files (*.stl *.obj *.dae);;All files (*.*)");
-  if (!path.isEmpty())
-    importResource("file://" + path.toStdString());
+  switch (ui_->shapes_combo_box->currentData().toInt())
```

Comment:
I guess it's clear from the usage, but I would not have guessed how this function works from just the name.

Reply:
Added a comment.